packages/hadron-build/lib/run.js

```diff
@@ -16,14 +16,14 @@ function getBinPath(cmd, fn) {
       return fn(err);
     }
 
-    fs.exists(bin, function(exists) {
-      if (!exists) {
-        return fn(new Error(format(
-          'Expected file for `%s` does not exist at `%s`',
-          cmd, bin)));
-      }
-      fn(null, bin);
-    });
+    // existsSync is at least not deprecated
+    const exists = fs.existsSync(bin);
+    if (!exists) {
+      return fn(new Error(format(
+        'Expected file for `%s` does not exist at `%s`',
+        cmd, bin)));
+    }
+    fn(null, bin);
```

Comment:
`access` is a recomended replacement for exists usage

```suggestion
    fs.access(bin, (err) => {
      if (err.code === "ENOENT") {
        return fn(
          new Error(
            format("Expected file for `%s` does not exist at `%s`", cmd, bin)
          )
        );
      }
      return fn(null, bin);
    });
```

Reply:
I'm actually just going to revert this commit because I fixed the string interpolation by working around webdriver/puppeteer overriding debug. No need for me to wade into hadron-build right now and I'm worried about test coverage there and so on.